libusb/hid.c

```diff
@@ -739,6 +739,105 @@ static uint16_t get_report_descriptor_size_from_interface_descriptors(const stru
 	return result;
 }
 
+static int is_xbox360(unsigned short vendor_id, const struct libusb_interface_descriptor *intf_desc)
+{
+	static const int XB360_IFACE_SUBCLASS = 93;
+	static const int XB360_IFACE_PROTOCOL = 1; /* Wired */
+	static const int XB360W_IFACE_PROTOCOL = 129; /* Wireless */
+	static const int SUPPORTED_VENDORS[] = {
```

Comment:
please don't use `UPPER_SNAKE_CASE` for named constants - it is generally error-prone in C/C++ due to fact that usually it is used for macros

here and below

Reply:
Would this be the preferred naming convention for these constants?
```
xb360_iface_subclass, ..., supported_vendors
```

Reply:
yes